File: pippip-native-ios/More/EditWhitelistCell.m
//
//  EditWhitelistCell.m
//  pippip-native-ios
//
//  Created by Steve Brenneis on 12/28/17.
//  Copyright © 2017 seComm. All rights reserved.
//

#import "EditWhitelistCell.h"

@interface EditWhitelistCell ()

@end

@implementation EditWhitelistCell

+ (MoreCellItem*)cellItem {

    MoreCellItem *item = [[MoreCellItem alloc] init];
    item.cellReuseId = @"EditWhitelistCell";
    item.cellHeight = 45.0;
    return item;

}

- (void)awakeFromNib {
    [super awakeFromNib];
    // Initialization code
}

- (void)setSelected:(BOOL)selected animated:(BOOL)animated {
    [super setSelected:selected animated:animated];

    // Configure the view for the selected state
}

@end

File: pippip-native-ios/More/EditWhitelistCell.swift
//
//  EditWhitelistCell.swift
//  pippip-native-ios
//
//  Created by Steve Brenneis on 3/22/18.
//  Copyright © 2018 seComm. All rights reserved.
//

import UIKit

class EditWhitelistCell: UITableViewCell {

    @objc class func cellItem() -> MoreCellItem {
        
        let item: MoreCellItem = MoreCellItem()
        item.cellHeight = 50.0
        item.cellReuseId = "EditWhitelistCell"
        return item
        
    }
    
    override func awakeFromNib() {
        super.awakeFromNib()
        // Initialization code
    }

    override func setSelected(_ selected: Bool, animated: Bool) {
        super.setSelected(selected, animated: animated)

        // Configure the view for the selected state
    }

}
